example-clients/jack_control

```diff
@@ -157,17 +157,16 @@ def main():
         sys.exit(0)
 
     bus = dbus.SessionBus()
-
-    controller = bus.get_object(service_name, "/org/jackaudio/Controller")
-    control_iface = dbus.Interface(controller, control_interface_name)
-    configure_iface = dbus.Interface(controller, configure_interface_name)
-
-    # check arguments
-    index = 1
-    while index < len(sys.argv):
-        arg = sys.argv[index]
-        index += 1
-        try:
+    try:
+        controller = bus.get_object(service_name, "/org/jackaudio/Controller")
+        control_iface = dbus.Interface(controller, control_interface_name)
+        configure_iface = dbus.Interface(controller, configure_interface_name)
```

Comment:
I think it is better to separate into 2 `try` blocks here.
1 for setting up dbus objects and interfaces, the other as it was before.

Reply:
fixed